Webrew.Data/Collections/ReviewCollection.cs

```diff
@@ -1,18 +1,26 @@
+using MongoDB.Bson;
 using MongoDB.Driver;
 using MongoDB.Driver.Linq;
 using System;
 using System.Collections.Generic;
 using System.Text;
+using System.Threading.Tasks;
 using Webrew.Common.Models;
 using Webrew.Data.Interfaces;
 
 namespace Webrew.Data.Collections
 {
-	public class ReviewCollection : WebrewCollection<Review>, IReviewCollection
-	{
-		private IMongoQueryable<Review> Reviews { get { return Collection.AsQueryable(); } }
-		public ReviewCollection(IDbClient client) : base(client)
-		{
-		}
-	}
+    public class ReviewCollection : WebrewCollection<Review>, IReviewCollection
+    {
+        private IMongoQueryable<Review> Reviews { get { return Collection.AsQueryable(); } }
+
+        public ReviewCollection(IDbClient client) : base(client)
+        {
+        }
+
+        public Task<Review> UpdateReview(ObjectId id, Review review)
+        {
+            throw new NotImplementedException();
```

Comment:
Whoops, did you not hook this to the collection?

Reply:
not quite sure, VS had yelled at me until I added that.